src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -0,0 +1,195 @@
+==== Housekeeping
+
+Notifications in the notification service have a global life span defined in the
+application.properties located in * `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/application.properties`
+Passed that time, they will be removed from the database.
+
+Can be defined:
+
+- The call frequency of the housekeeping defined by a CRON expression:
+----
+# This cron expression determines the housekeeping call frequency.
+# Default value is 10 minutes: this will invoke the housekeeping mechanism
+# to remove every notifications whose scheduled removal time is up.
+notifications.housekeeping.cronexpression=*/10 * * * *
+----
+
+- The life span of the notifications in seconds:
+----
+# Automatic remove notification delay (in seconds).
+# The time at which notifications will be stored in the database
+# Set this time to 0 if you don't want the notifications to be removed.
+# Default value is 604800 seconds = 1 week
+notifications.housekeeping.removedelay=604800
+----
+
+[[_notification_methods]]
+==== Notification methods
+
+[[_email_notification_method]]
+===== Email
+
+Email notification method uses SMTP protocol to send emails, thus the notification
+service needs to have knowledge and connect to a SMTP server to use it.
+Relevant data is specified in the properties file located
+* `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/email.properties`
+
+By default Transport Layer Security is active, change to false to deactivate it:
+----
+mail.smtp.starttls.enable=true
+----
+Authentication to the SMTP server is enabled by default.
+----
+mail.smtp.auth=true
+----
+
+Provide the server port used to connect
+----
+mail.smtp.port=
+----
+
+The credentials used to connect
+----
+mail.smtp.username=
+mail.smtp.password=
+----
+
+The host of the SMTP server
+----
+mail.smtp.host=
+----
+
+The email used to broadcast the emails is defined in the application.properties file,
+go to <<Housekeeping>> to see where it is located.
+The email will be displayed in the "From" field of the sent emails
+----
+notification.email.from=
+----
+
+[[_third_party_notification_method]]
+===== Third party
+
+====== Configuration
+
+The Third party notification method executes a script when an event occurs.
+Location of the script is specified either via a path, relative to the Scheduler or absolute.
+
+----
+notification.script.path=dist/war/notification-service/WEB-INF/classes/exampleScript.sh
+----
+
+Or a URL
+
+----
+notification.script.url=
+----
+
+Note that to the URL property will be used by the <<_glossary_notification_service,*Notification Service*>> only if the path is not provided.
```

Comment:
you should use a asciidoc IMPORTANT: tag (or something else)

Note that to the `notification.script.url` property  
Otherwise, the `notification.script.url` property  is ignored...